app/views/groups/users.html.erb

```diff
@@ -1,14 +1,26 @@
+<div class="sml-title-purple-align-right">
+  directory
+</div>
+
+<div class="two-color-sentence-align-right">
+  <%= @group.name.downcase %>
+</div>
 
-<h1>Welcome to <%= @group.name %></h1>
-<div class="mentor-list">
+<div class="group-list">
```

Comment:
Awesome work Matt. Happy for this to change from mentor-list to group-list, because it's a more accurate name. However, I saw that Yumi called the class mentor-list and haven't seen yet that the name has been updated in the scss file. No stress for now, but when you have a minute can you please update it there as well? Otherwise the styling won't apply here. Thanks!